web/src/hooks/redux.js

```diff
@@ -53,12 +53,7 @@ export function useCurrentZoneData() {
     if (!zoneId || !grid || !zoneHistory) {
       return null;
     }
-    if (zoneTimeIndex === null) {
-      // Return latest history data unless latest history does not correspond to the current grid datetime
-      return (zoneHistory[zoneHistory.length - 1].stateDatetime === grid.datetime
-        ? zoneHistory[zoneHistory.length - 1]
-        : null);
-    }
+
```

Comment:
What will happen if if `zoneTimeIndex` is `null` now?

An alternative could be return the latest one and ignoring that it might be out of date (as that will be handles with historical view soon anyway)
```suggestion
if (zoneTimeIndex === null) {
      return zoneHistory.at(-1);
    }
```